solidity/random-beacon/contracts/libraries/Relay.sol

```diff
@@ -189,17 +181,15 @@ library Relay {
         self.relayRequestFee = uint96(newRelayRequestFee);
     }
 
-    /// @notice Set relayEntrySubmissionEligibilityDelay parameter.
-    /// @param newRelayEntrySubmissionEligibilityDelay New value of the parameter.
-    function setRelayEntrySubmissionEligibilityDelay(
+    /// @notice Set relayEntrySoftTimeout parameter.
+    /// @param newRelayEntrySoftTimeout New value of the parameter.
+    function setRelayEntrySoftTimeout(
         Data storage self,
-        uint256 newRelayEntrySubmissionEligibilityDelay
+        uint256 newRelayEntrySoftTimeout
     ) internal {
         require(!isRequestInProgress(self), "Relay request in progress");
 
-        self.relayEntrySubmissionEligibilityDelay = uint32(
-            newRelayEntrySubmissionEligibilityDelay
-        );
+        self.relayEntrySoftTimeout = uint32(newRelayEntrySoftTimeout);
```

Comment:
Thoughts on removing casting by either defining soft/hard timeouts as `uint256` or passing `uint32`?

Reply:
I'd leave it as is in this PR for consistency. We should consider updating it in a separate PR to determine how changes will impact gas costs.